ghga_connector/core/api_calls.py

```diff
@@ -145,7 +145,7 @@ def download_api_call(api_url: str, file_id: str) -> Tuple[Optional[str], int]:
         if "retry-after" not in headers:
             raise RetryTimeExpectedError(url)
 
-        return None, int(headers["retry-after"])
+        return None, int(headers["retry-after"]), 0
```

Comment:
This is a bit confusing to read, together with L.163.
Had to jump between those and the call site to figure out, what's going on
Maybe alias the null/default case for better readability?

Reply:
In the end I decided to us the constant NO_FILE_SIZE = 0, in order to keep the type with int. Converting the int to Optional[int] opend problems on the receiving end and I would have had to convert None back to 0 anyways.